airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -144,14 +145,16 @@ class DbtCloudHook(HttpHook):
     def get_ui_field_behaviour() -> Dict[str, Any]:
         """Builds custom field behavior for the dbt Cloud connection form in the Airflow UI."""
         return {
-            "hidden_fields": ["host", "port", "schema", "extra"],
-            "relabeling": {"login": "Account ID", "password": "API Token"},
+            "hidden_fields": ["host", "port", "extra"],
+            "relabeling": {"login": "Account ID", "password": "API Token", "schema": "Tenant (Optional)"},
```

Comment:
```suggestion
            "relabeling": {"login": "Account ID", "password": "API Token", "schema": "Tenant"},
            "placeholders": {"schema": "Defaults to 'cloud'."},
```
Not a wildly-strong opinion and nit-adjacent, but it might be useful to add a placeholder of something like "Defaults to 'cloud'." rather than having "(Optional)" in the field name of the form. WDYT?

Also, it doesn't have to be in this PR, but definitely a fast-follow, updating the [connection doc](https://airflow.apache.org/docs/apache-airflow-providers-dbt-cloud/stable/connections.html) with this new `tenant` parameter would be great.